.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
+  // Ignore unknown CSS at rules for Tailwind CSS
+  "css.lint.unknownAtRules": "ignore",
+  // Enable Stylelint in javascript and typescriptreact (for CSS-in-JS) and scss
+  "stylelint.validate": [
+    "css",
+    "scss",
+    "postcss",
+    "javascript",
+    "typescriptreact"
+  ]
+}
```

Comment:
another error caused by the broken project type detection in `eslint-config-upleveled` - this VS Code `settings.json` should only be installed in CRA and Next.js apps:

- https://github.com/upleveled/eslint-config-upleveled/blob/main/templates/next-js-postgresql/.vscode/settings.json

Reply:
Removed the `settings.json` file in this commit
[9abb807](https://github.com/upleveled/expo-example-spring-2024-atvie/pull/4/commits/9abb8070e8c28bee99bd553044562f3674bf6f7d)